### PR TITLE
#2053 Unify collection detail and collections view with $primary

### LINKED ISSUE #2053: Unify collection detail and collections view with $primary
### At collection detail 
- Make tab line pink, on selected it can be under text white and selected text can be kept pink
- filter border $primary
- search box $primary

image

Test at https://kodadot.xyz/rmrk/collection/d23ec8e0b831b65917-LTIDO?search=&listed=false
### Collection
- same $primary for filter & search bar

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Do a quick check before the merge.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2053
- [x] Since it is common, this also got applied to artist profile

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI, a screenshot is best to understand changes for others.
Screenshot 2022-01-26 at 2 56 56 PM
Screenshot 2022-01-26 at 2 57 09 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
I don't understand why we need to change these two searchbar borders in particular.
I feel like there is too much pink in the design.

Other solution could be to add a `card` border like in Gallery Search
Screenshot 2022-01-26 at 11-15-28 Low minting fees and carbonless NFTs

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue [hunk 1]
```diff
@@ -33,8 +33,13 @@ export default class SearchSortDropdown extends Vue {
...
 </script>
 
-<style scoped lang="scss">
+<style lang="scss">
+@import '@/styles/variables';
+
 .select-dropdown {
+  select {
+    border: 1px solid $primary!important;
+  }
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:42]: @roiLeo doing this global would apply this everywhere, in the ticket its only mentioned for collection though

File: components/rmrk/Gallery/Search/SearchBarCollection.vue [hunk 2]
```diff
@@ -144,12 +145,19 @@ export default class SearchBar extends Vue {
...
   .is-grouped-multiline {
     flex-wrap: initial !important;
     @media screen and (max-width: 768px) {
       flex-wrap: wrap !important;
     }
   }
+  .input-search {
+    input {
+      border: 1px solid $primary!important;
+    }
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchBarCollection.vue:160]: @roiLeo doing this global would apply this everywhere, in the ticket its only mentioned for collection though

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:42]: if this is inteded could you please move this outside component as we use `select` in other components

[the remark this answers is not in the document]
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:42]: yes so we use this in collections, collection item and artist profile, which I think is the intended behaviour, so it should be fine right?

**Comment by yangwao** [comment]
> I don't understand why we need to change these two searchbar borders in particular. I feel like there is too much pink in the design.

The goal is to have prepared stuff for changing Themes #153 so having primary will make it easier I guess.
Plus we would like to unify in one colour. I mean, a good option would be to go for some gradient as I sense that control panels/buttons/inputs should be distinguished somehow from non-actionable stuff

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:42]: Nevermind, I thought that `.select-dropdown` was a bulma class.
Now the thing that concerns me is, I can see there is already a state (`:active`) where the primary color is applied on borders, so this change will introduce a different use of the select component.

[the remark this answers is not in the document]
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:42]: yeah so currently everything is pink, even the active one, which is intended I think

**Comment by roiLeo** [comment]
Personal preference I find the tabs really weird, my eye is used to having a primary color (& border bottom) when component is active.
Plus there is a risk of introducing unexpected behaviour like hover state:
#### Now
Screenshot 2022-01-26 at 15-03-24 Gyp 1 DAO
#### Before
Screenshot 2022-01-26 at 15-03-03 Cat DAO

**Comment by prachi00** [comment]
@yangwao @roiLeo let me know if I should revert some changes in this or implement in a different way?

**Comment by yangwao** [comment]
> Personal preference I find the tabs really weird, my eyes are used to have a primary color (& border bottom) when component is active. (might come from material design) Plus there is a risk of introducing unexpected behaviour like hover state:
> 
> #### Now
> Screenshot 2022-01-26 at 15-03-24 Gyp 1 DAO
> 
> #### Before
> Screenshot 2022-01-26 at 15-03-03 Cat DAO

yeah, that's a good perspective. I was struggling that active state &c should be bit redesigned how we using it and bit unifying it as whole

@prachi00 let's keep tabs styling to the previous state as there will be too much pink where @roiLeo is definitely right :)

**Comment by prachi00** [comment]
@yangwao have reverted the colour changes for tab

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 167.63 USD/KSM ~ 0.597 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x333be4aca31e5eacbe5af3a079e6b56388400cd6959c31a3e104776baf20f693](https://kusama.subscan.io/extrinsic/0x333be4aca31e5eacbe5af3a079e6b56388400cd6959c31a3e104776baf20f693)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 157.66 USD/KSM ~ 0.634 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xc321cad58c3cb0182b479da24318a0b4917b07a62b8fd52eb38dd9d853dfa661](https://kusama.subscan.io/extrinsic/0xc321cad58c3cb0182b479da24318a0b4917b07a62b8fd52eb38dd9d853dfa661)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**